### ISSUE TITLE
The string of time :-)

### ISSUE THREAD

**Issue by marceltaeumel** [hpi-swa/Squot#224]
I would be nice to see a relative string representation of timestamps:

![squot-timestamp](https://user-images.githubusercontent.com/5765308/55159262-e740ba80-5160-11e9-903e-2244be29ea37.png)

This code might help:
[ViLabelStringService-doDateAndTime.txt](https://github.com/hpi-swa/Squot/files/3017979/ViLabelStringService-doDateAndTime.txt)

**Comment by j4yk** [comment]
In my experience, the timestamps are important for active projects even when they are not from today. And to omit the time if it is the only version on a particular day, you would have to look at the surrounding versions.

**Comment by marceltaeumel** [comment]
That's right, you could add the time there, too. Yet, a quick peek to see "Yesterday" or "Thursday" (if the same week) instead of "Jun 12 1999" helps a lot. Overall, this should be subject to configuration and not hard-coded.

**Comment by j4yk** [comment]
I'd rather prefer to not introduce so many customization options if it is not absolutely necessary. But I see the benefit of having "Yesterday" or maybe "*x* d ago <time>" (up to a week) there.
